goleveldb.go

```diff
@@ -24,7 +25,10 @@ type GoLevelDB struct {
 var _ DB = (*GoLevelDB)(nil)
 
 func NewGoLevelDB(name string, dir string) (*GoLevelDB, error) {
-	return NewGoLevelDBWithOpts(name, dir, nil)
+	options := &opt.Options{
+		Filter: filter.NewBloomFilter(10), // by default, goleveldb doesn't use a bloom filter.
```

Comment:
should we also disable seek compaction?

Reply:
will look it up this week.  Maybe we start here, and start to feed things into the iavl benchmarks-- they work now